packages/react/src/components/UserInformation/UserInformation.js

```diff
@@ -0,0 +1,308 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { css } from '@emotion/react';
+import Sidebar from '../Sidebar/Sidebar';
+import RCContext from '../../context/RCInstance';
+import { useUserStore } from '../../store';
+import { Box } from '../Box';
+import { Avatar } from '../Avatar';
+import { Icon } from '../Icon';
+import { Throbber } from '../Throbber';
+import { appendClassNames } from '../../lib/appendClassNames';
+
+const userSidebarCss = css`
+  padding: 0 1rem 1rem;
+  margin: 0 auto;
+`;
+
+const userRoleCss = css`
+  background-color: #cbced1;
+  letter-spacing: 0rem;
+  font-size: 0.75rem;
+  padding: 0 0.25rem;
+  margin: 0 0.1rem;
+  border-radius: 2px;
+  font-weight: 700;
+  line-height: 1.25rem;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  color: #2f343d;
+`;
+
+const UserInformation = () => {
+  const { RCInstance } = useContext(RCContext);
+
+  const [currentUserInfo, setCurrentUserInfo] = useState({});
+  const [isUserInfoFetched, setIsUserInfoFetched] = useState(false);
+  const setShowCurrentUserInfo = useUserStore(
+    (state) => state.setShowCurrentUserInfo
+  );
+  const currentUser = useUserStore((state) => state.currentUser);
+  const authenticatedUserRoles = useUserStore((state) => state.roles);
+  const authenticatedUserId = useUserStore((state) => state.userId);
+  const isAdmin = authenticatedUserRoles.includes('admin');
+
+  const getUserAvatarUrl = (username) => {
+    const host = RCInstance.getHost();
+    const URL = `${host}/avatar/${username}`;
+    return URL;
+  };
+
+  const formatTimestamp = (timestamp) => {
```

Comment:
can we please move this to a separate file inside https://github.com/RocketChat/EmbeddedChat/tree/develop/packages/react/src/lib?

Reply:
@sidmohanty11 
Modified the code as required.